tools/depends/configure.ac

```diff
@@ -347,12 +371,16 @@ case $host in
         # setup which sdk to use
         if test "$use_platform" = "tvos"; then
           use_cpu=arm64
+          use_cmake_cpu=aarch64
           target_platform=appletvos
           platform_min_version="$target_platform-version-min=11.0"
         else
           # setup which cpu to use
           if test "x$use_cpu" = "xauto"; then
             use_cpu=armv7
+            use_cmake_cpu=armv7
+          else
+            use_cmake_cpu=$use_cpu
```

Comment:
Becomes for iOS 64bit always the `--with-cpu` given by configure? 
Was confused that there no `armv8` or something else becomes defined.

Reply:
Seen the `--with-cpu=arm64` becomes used, I add a check for this name and set it then to `aarch64` for `use_cmake_cpu`, any different `--with-cpu=` there possible?